utils/apiPerformance.cjs

```diff
@@ -6,7 +6,7 @@ const path = require('path')
 const url = require('url')
 const { exec } = require("child_process");
 
-let baseContext = '/cli-adapter-api-testing/v1';
+let baseContext = '/cli-adapter-api-performance/v1';
```

Comment:
why there is a change in the context path? wasn't it the same from before or is it a new change?

Reply:
it is new change